Gems/AudioSystem/Code/Source/Editor/AudioControl.h

```diff
@@ -25,11 +25,11 @@ namespace AudioControls
     {
         SRawConnectionData(AZ::rapidxml::xml_node<char>* node, bool isValid)
         {
-            m_xmlNode = DeepCopyNode(node);
+            m_xmlNode = AZStd::move(DeepCopyNode(node));
             m_isValid = isValid;
         }
 
-        AZ::rapidxml::xml_node<char>* m_xmlNode{ nullptr };
+        AZStd::unique_ptr<AZ::rapidxml::xml_node<char>> m_xmlNode{};
```

Comment:
I am assuming this was leaking memory before?

Reply:
It wasn't great before, so this is much better.  I think the rapidxml memory_pool probably handles all cleanup when it gets destroyed.  Not sure about how it handles the situation where the pool is full and it overflows to using dynamic allocations.